Application/Features/BookingFeature/BookingProfile.cs

```diff
@@ -0,0 +1,24 @@
+﻿using Application.Features.BookingFeature.Queries;
+using AutoMapper;
+using Domain.Models;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Application.Features.BookingFeature
+{
+    public class BookingProfile : Profile
+    {
+        public BookingProfile()
+        {
+            CreateMap<Booking, BookingDto>();
+            CreateMap<Booking, GetBookingByIdQueryResponse>();
+            CreateMap<User, UserDto>()
```

Comment:
wrong, you should include UserName into you bookingDro or bookingDtoResponse and map it to there, no direct to user